projects.md

```diff
@@ -0,0 +1,160 @@
+
+**Status:** Always in Beta  
+This project will keep being actively developed to support ongoing research work and experimentation. It has been forked by many groups for their own single-board computer mesh network projects.
+
+**Skills**  
+Single-board computers, shell scripts, technical documentation, Debian systems, network configuration, test automation on ARM devices, cjdns, peer-to-peer applications.
+
+[Prototype Node GitHub page](https://github.com/tomeshnet/prototype-cjdns-pi)
+
+### Mesh Orange
+
+Mesh Orange is a small Debian-based operating system image that runs on single-board computers (SBCs), turning them into easily configurable mesh nodes.
+
+The custom build system allows us to create custom images for workshops, and we expect this to become the base operating system for production mesh nodes, with features such as over-the-air updates and reliable long-term operation.
+
+**Status:** Alpha  
+Mesh Orange gets new features as new use cases come up. Raspberry Pi and Orange Pi Zero boards are supported (board support is not as extensive as with  Prototype Node).
+
+**Skills**  
+Single-board computers, shell scripts, technical documentation, Debian systems, network configuration, test automation on ARM devices, building operating systems for embedded targets.
+
+[Mesh Orange GitHub page](https://github.com/tomeshnet/mesh-orange)
+
+### Mapping & Monitoring Tools
+
+Software that runs on a Raspberry Pi and can be provisioned for cloud providers to monitor real-time status of the mesh network.
+
+The Monitored data includes geolocation mapping on a web interface with real-time information reported from nodes, and plotting logical mesh topology on a geographical map. This will be packaged in a way that makes it very easy for other mesh networks to adapt to their own monitoring needs.
+
+**Status:** Concept  
+Lead required to further develop proof-of-concept work.
+
+**Skills**  
+Devops, Terraform, Docker, devops, Prometheus, mapping, data visualization, networking, web design, Google Maps, Open Street Maps, shell scripts, python, cjdns.
+
+[Monitoring GitHub page](https://github.com/tomeshnet/monitoring)
+
+---
+Collaboration and Outreach Tools
+
+---
+
+### Toronto Mesh Website
+
+The primary digital space for people to learn about Toronto Mesh. It must represent the current focus of our organization and remain accessible to diverse audiences.
+
+**Status:** Production (live)  
+Need to be spun up with provisioning tools and served over cjdns, IPFS, DAT, etc.
+
+**Skills**  
+Web design, user research, javascript, Terraform, peer-to-peer hosting.
+
+[Toronto Mesh Website GitHub page](https://github.com/tomeshnet/tomesh.net)
+
+### Meshbot
+
+The chat bot our Matrix chat uses to send event reminders and answer basic questions about the mesh.
+
+**Status:** Production
+
+**Skills**  
+Programming, user research, technical documentation
+
+[Meshbot GitHub page](https://github.com/tomeshnet/meshbot)
+
+### Mesh Community Assets
```

Comment:
I'm not sure if this is a meaningful name for what this project is doing? Partially this is maintaining the infra powering tomesh right? But in a replicatable way with mutual solidarity in mind?

Reply:
Agreed, I changed it to "Toronto Mesh Infrastructure"